src/processors/default/addMimeTypeToProcessedTracks.ts

```diff
@@ -0,0 +1,33 @@
+import { Table } from '../../db/db';
+import { missingMimeType } from '../../triggers/missing';
+import { Clients, Processor } from '../../types/processor';
+import { ProcessedTrack, SourceIPFS } from '../../types/track';
+import { rollPromises } from '../../utils/rollingPromises';
+
+export const addMimeTypeToProcessedTracks: (source: SourceIPFS) => Processor =
+  (source) => ({
+    name: `addLossy${source}MimeTypeToProcessedTracks`,
+    trigger: missingMimeType(source),
+    processorFunction: async (processedTracks: ProcessedTrack[], clients: Clients) => {
+      console.log(`Fetching processed track ${source} mime types`);
+
+      const updatedMimeTypes = async (processedTrack: any) => {
+        const id = processedTrack.id;
+        const ipfsHash = processedTrack[`lossy${source}IPFSHash`];
+        const response = await clients.axios.head(`${process.env.IPFS_ENDPOINT}${ipfsHash}`, { timeout: parseInt(process.env.METADATA_REQUEST_TIMEOUT!) })
+
+        const result: any = { id: id }
+        result[`lossy${source}MimeType`] = response.headers['content-type'];
+        return result;
+      }
+
+      const results = await rollPromises<ProcessedTrack, any, void>(
```

Comment:
need error handling, and probably could use same error/retry functionality that we have for uploading ipfs files? else this could retry indefinitely for a broken file/link?